internal/cli/artifact_build.go

```diff
@@ -73,7 +73,7 @@ func (c *ArtifactBuildCommand) Synopsis() string {
 func (c *ArtifactBuildCommand) Help() string {
 	return formatHelp(`
 Usage: waypoint artifact build [options]
-Alias: waypoint build
+Alias: waypoint build [options]
```

Comment:
As it stands, Usage and Alias show up on the same line:

<img width="701" alt="Screen Shot 2021-08-10 at 9 41 26 AM" src="https://user-images.githubusercontent.com/8404559/128877537-91fa9020-b307-49cd-9945-9744859718b6.png">

I think we probably want those on separate lines, like the CLI does. We'd need a little more regex then though - just adding a newline causes the alias command to lose code block formatting.

Reply:
Hm, the docs gen appears to be working for me with just adding a newline like so
```
	fmt.Fprintf(w, fmt.Sprintf("\nAlias: `waypoint %s`\n", aliasMatch[1]))
```

Reply:
Oh nice! I only tried adding the newline in the help string.